modules/osdk-ansible-metrics.adoc

```diff
@@ -0,0 +1,225 @@
+[%collapsible]
+====
+[source,yaml]
+----
+---
+# tasks file for Memcached
+- name: start k8sstatus
+  k8s:
+    definition:
+      kind: Deployment
+      apiVersion: apps/v1
+      metadata:
+        name: '{{ ansible_operator_meta.name }}-memcached'
+        namespace: '{{ ansible_operator_meta.namespace }}'
+      spec:
+        replicas: "{{size}}"
+        selector:
+          matchLabels:
+            app: memcached
+        template:
+          metadata:
+            labels:
+              app: memcached
+          spec:
+            containers:
+            - name: memcached
+              command:
+              - memcached
+              - -m=64
+              - -o
+              - modern
+              - -v
+              image: "docker.io/memcached:1.4.36-alpine"
+              ports:
+                - containerPort: 11211
+
+- osdk_metric:
+    name: my_thing_counter
+    description: This metric counts things
+    counter: {}
+
+- osdk_metric:
+    name: my_counter_metric
+    description: Add 3.14 to the counter
+    counter:
+      increment: yes
+
+- osdk_metric:
+    name: my_gauge_metric
+    description: Create my gauge and set it to 2.
+    gauge:
+      set: 2
+
+- osdk_metric:
+    name: my_histogram_metric
+    description: Observe my histogram
+    histogram:
+      observe: 2
+
+- osdk_metric:
+    name: my_summary_metric
+    description: Observe my summary
+    summary:
+      observe: 2
+----
+====
+
+.Verification
+
+. Run your Operator on a cluster. For example, to use the "run as a deployment" method:
+
+
+.. Build the Operator image and push it to a registry:
++
+[source,terminal]
+----
+$ make docker-build docker-push IMG=<registry>/<user>/<image_name>:<tag>
+----
+
+.. Install the Operator on a cluster:
++
+[source,terminal]
+----
+$ make install
+----
+
+.. Deploy the Operator:
++
+[source,terminal]
+----
+$ make deploy IMG=<registry>/<user>/<image_name>:<tag>
+----
+
+. Create a `Testmetrics` custom resource (CR):
+
+.. Define the CR spec:
++
+.Example `config/samples/metrics_v1_testmetrics.yaml` file
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: metrics.testmetrics.com/v1
+kind: Testmetrics
+metadata:
+  name: testmetrics-sample
+spec:
+  size: 1
+----
+====
+
+.. Create the object:
++
+[source,terminal]
+----
+$ oc create -f config/samples/metrics_v1_testmetrics.yaml
+----
+
+. Get the custom metrics token:
++
+[source,terminal]
+----
+$ token=`oc sa get-token prometheus-k8s -n openshift-monitoring`
+----
+
+. Get the endpoint details:
++
+[source,terminal]
+----
+$ oc get ep
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                                ENDPOINTS          AGE
+ansiblemetrics-controller-manager-metrics-service   10.129.2.70:8443   150m
+----
+
+. Check the metrics values:
+
+.. Check the `my_counter_metric` value:
++
+[source,terminal]
+----
+$ oc exec ansiblemetrics-controller-manager-58dd4f465-ws8j8 -- curl -k -H "Authoriza
```

Comment:
Doesn't this need to be replaced with sample text?
Same question for lines 186-187, 202-203, 216-217.

Reply:
Changed `-58dd4f465-ws8j8` to `-<id>` for any of the pod name references.